test/index.spec.js

```diff
@@ -118,6 +119,17 @@ function runTestWithConfig (testConfig, initialStoreState, label) {
         }
       })
 
+      it('should record non state changing actions when filter is not set', () => {
+        let dummyState = mockUndoableReducer(incrementedState, { type: 'DUMMY' })
+        if (testConfig.FOR_TEST_ONLY_distinctState) {
+          expect(dummyState).to.deep.equal(incrementedState)
+        } else {
+          expect(dummyState.present).to.deep.equal(incrementedState.present)
+          expect(dummyState.past).to.deep.equal([...incrementedState.past, incrementedState.present])
+          expect(dummyState.future).to.deep.equal(incrementedState.future)
+        }
+      })
+
```

Comment:
And a test for distinct state filter, will increase coverage to 100% when the other is merged :)